amp/WEB-INF/src/org/digijava/module/aim/util/ProgramUtil.java

```diff
@@ -148,17 +148,16 @@ public static AmpTheme getThemeByCode(String code) {
         }
 
 
-        public static AmpTheme getTheme(String name) {
+        public static AmpTheme getTheme(String id) {
```

Comment:
you are changing the search variable and its being used in 
AddTheme.checkExisting#151